Create Blog Page with Post About Missed Fines

vkoves opened the pull request:
# Description

Adds a new blog page with a new article about how the city missed out in $30 million in fines by not enforcing the energy benchmarking ordinance.

| Blog Page | Post Top |
| --- | --- |
| ![Screenshot from 2024-02-27 23-57-05](https://github.com/vkoves/electrify-chicago/assets/3187531/ab2d8afc-4efb-4ff6-b67d-e01e65d0b6cc) | ![Screenshot from 2024-02-27 23-57-26](https://github.com/vkoves/electrify-chicago/assets/3187531/e1c33db6-a21c-4865-af2d-5d58d0a7c34f) |

Resolves #75

# Testing Instructions

Viewed the new blog page and post on desktop and mobile and ran AxeDevTools to confirm it has no accessibility issues.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

<!-- PR template modified from: https://embeddedartistry.com/blog/2017/08/04/a-github-pull-request-template-for-your-projects/ -->

vkoves commented:
Suggestion in Slack from Soren:

> Parachuting in to nitpick that I think in several places where you use the word "buildings" alone to describe the party that is required to report, it may be better to use "building owners". It's like the "car" vs. "driver" distinction in traffic crash reporting.

derekeder commented on src/pages/blog/MillionsInMissedFines.vue:
```diff
@@ -0,0 +1,179 @@
+      >
+
+      <p class="caption">
+        The full data portal screenshot, and a zoomed in view of the query result count (
```
this line seems to be not rendering properly 

![Screenshot 2024-02-28 at 4 30 14 PM](https://github.com/vkoves/electrify-chicago/assets/919583/373e962e-594f-4287-b7a4-ed9dc6cd8997)

derekeder commented:
this is great - hopefully it will turn a few heads. do you plan to pitch to a publication? block club, trib or sun-times might be interested.

a few notes:

- might be worth updating the home page to feature this blog post if you want more people to find/read it
- shorter version of the headline reads a little better: "City Of Chicago Failed to Collect $30 Million In Potential Fines from Building Benchmarking Ordinance"
- add a way to contact you for follow-up information

vkoves commented:
@derekeder - great suggestions, I've added my external contact email to the post and added a new panel to the homepage:

![Screenshot from 2024-02-28 17-00-07](https://github.com/vkoves/electrify-chicago/assets/3187531/a32fa1f7-a7a0-44de-ae1c-d2ac07800803)

Thoughts?

derekeder commented:
nice. i think the two alerts are competing with each other with color a bit. might look better if they were stacked vertically instead of horizontal too.

vkoves commented:
@derekeder - I did that before, but then you have to scroll down more on desktop view the actual table. I could make the legislative update background white, perhaps?

derekeder commented:
yah white background would help i think

vkoves commented:
Merging this in since folks have looked at it and it's not changing data
